notebooks/community/ml_ops/stage6/get_started_with_custom_image_model_batch.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>I think there are public docs explaining these input types. Can we put a summary and refer to the main docs by linking them here instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/915/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>The web doc is out of date. Its a mismash of legacy (CAIP) and Vertex. TW are assigned to update.</p><p> </p><p>This comes from internal docs on SWE team. The closest we have to an accurate description.</p>

Reply:
Oh makes sense! Thanks!